assets/blocks/contact-teacher-block/contact-teacher.scss

```diff
@@ -1,12 +1,117 @@
-.sensei-contact-teacher-form {
-	margin-top: 1em;
-	margin-bottom: 1em;
+.sensei-course-theme__frame, * {
+	.sensei-contact-teacher-form {
+		position: relative;
```

Comment:
Had to do some css dancing to make sure the contact-teacher block is styled the same inside `.sensei-course-theme__frame` and outside it.